### PR TITLE
redirect cron stdout to docker

### PR DESCRIPTION
**Pull request by SamEureka**
@chrisys & @catt-tung ,

Figured I would just clear all the issues while I was in the repo. 🤣 

Just needed to redirect the cron stdout to the docker proc. Also redirected stderr to stdout just in case. 

resolves balenalabs/inkyshot#17

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ python /usr/app/update-display.py
 export -p > /usr/app/env.sh
 
 # Add the job to the crontab using update_hour var, defaulting to 9 AM
-(echo "0 ${UPDATE_HOUR:-9} * * * /usr/app/run-update.sh") | crontab -
+(echo "0 ${UPDATE_HOUR:-9} * * * * /usr/app/run-update.sh > /proc/1/fd/1 2>&1") | crontab -
```
Comment (chrisys) [inline start.sh:15]: I noticed you added an additional asterisk to the cron expression here - was that intentional?

**Review by chrisys — APPROVED** [review]

**Comment by SamEureka** [comment]
@chrisys 
Sorry! I found a problem in the crontab entry. I had added an extra * somehow. Fixed and squashed the commits. You'll need to review again.

**Comment by chrisys** [comment]
@SamEureka no worries, thanks again for this batch of contributions!

**Comment by SamEureka** [comment]
adventure_time-high_five

**Pull request merged**